src/w3c/word-usage-example.js

```diff
@@ -0,0 +1,62 @@
+import TextQuoteSelector from './text-quote-selector'
+
+export default class WordUsageExample extends TextQuoteSelector {
+  createContext () {
+    return null // not implemented in the current child-class
+  }
+
+  /**
+  * Creates WordUsageExample object from jsonObj, homonym, author, textWork and link from the adapter config
+  * @param {Object} jsonObj - json object from concordance api
+  * @param {Homonym} homonym - source homonym object
+  * @param {Author} author - source author object, could be undefined
+  * @param {TextWork} textWork - source textWork object, could be undefined
+  * @param {String} sourceLink - sourceTextUrl from the adapter config file
+  * @returns {WordUsageExample}
+  */
+  static readObject (jsonObj, homonym, author, textWork, sourceLink) {
```

Comment:
I think this also it too tightly coupled to the PHI concordance service's output format. Please parse the JSON of the service output there, and explicitly construct a WordUsageExample .  See more in the client-adapters PR.

Reply:
Done